src/link-handler.js

```diff
@@ -135,7 +139,6 @@ export class DefaultLinkHandler extends LinkHandler {
 
     return !targetWindow ||
       targetWindow === win.name ||
-      targetWindow === '_self' ||
-      (targetWindow === 'top' && win === win.top);
```

Comment:
Why did you delete this line of code?

Reply:
I mentioned this in my comment. I removed `(targetWindow === 'top' && win === win.top)` because it should have been `targetWindow === '_top'` and I didn't want to fix it in case anyone is using `target="_top"` as a workaround for this problem.

Reply:
I don't follow, though. It should be _top, but that wouldn't work because this code is wrong. So maybe someone is using target="top" instead right? And then this would break that?

Reply:
No, sorry, I meant that perhaps someone is using target="_top" to prevent the router from processing the link, and fixing the code would break that. For example the original poster of <https://github.com/aurelia/router/issues/428> did this. But it's completely up to you. If you do include that line, I suggest for completeness sake including another line with parent replacing top.